reconcile/change_owners/change_log_tracking.py

```diff
@@ -78,6 +83,7 @@ def run(
                 ChangeLogItem(**i)  # type: ignore[arg-type]
                 for i in existing_change_log.items
             ]
+        gl = init_gitlab(self.params.gitlab_project_id)
```

Comment:
using gitlab will probably be beneficial quickly. relying on the s3 object date is not stable, since we don't know if we will update the objects out of band in the future. i'll add a cleanup